lib/chef/provider/package/cab.rb

```diff
@@ -45,7 +45,7 @@ def remove_package(name, version)
         end
 
         def dism_command(command)
-          shellout = Mixlib::ShellOut.new("dism.exe /Online #{command} /NoRestart", { :timeout => @new_resource.timeout })
+          shellout = Mixlib::ShellOut.new("dism.exe /Online /English #{command} /NoRestart", { :timeout => @new_resource.timeout })
```

Comment:
If I change shellout ot using shell_out() like this one
```ruby
shellout = shell_out("dism.exe /Online #{command} /NoRestart", { :timeout => @new_resource.timeout })
```
I got raise erro
`Chef::Exceptions::Package
           -------------------------
           Found multiple packages installed matching name WinEmb-Enhanced-Write-Filter, found: 2 matches
`
https://github.com/jugatsu/chef/blob/7ceaff472def765635af448eb165ebf7a3e74399/lib/chef/provider/package/cab.rb#L69